packages/botonic-react/src/components/markdown.js

```diff
@@ -6,29 +6,47 @@ export const ESCAPED_LINE_BREAK = '&lt;br&gt;'
 const ESCAPED_LINE_BREAK_REGEX = new RegExp(ESCAPED_LINE_BREAK, 'g')
 const isLineBreakElement = element => element.type === 'br'
 
-const configureMarkdownRenderer = () => {
-  const md = new MarkdownIt({
-    html: true,
-    linkify: true,
-    typographer: true,
-  })
+const withLinksTarget = (renderer, target = '_blank') => {
   // Support opening links in new tabs: https://github.com/markdown-it/markdown-it/blob/master/docs/architecture.md#renderer
-  const renderer =
-    md.renderer.rules.link_open ||
+  const newRenderer =
```

Comment:
is there a better name than newRenderer? "new" is not so expressive. 
does mdRenderer or withoutMdRenderer make sense?

Reply:
this is only in the local scope of `withLinksTarget` to differentiate it from the one received in the first argument. I think that the diff lines make it confusing maybe.
```
const withLinksTarget = (renderer, target = '_blank') => {
  // Support opening links in new tabs: https://github.com/markdown-it/markdown-it/blob/master/docs/architecture.md#renderer
  const newRenderer =
    renderer.renderer.rules.link_open ||
    function (tokens, idx, options, env, self) {
      return self.renderToken(tokens, idx, options)
    }
  renderer.renderer.rules.link_open = function (
    tokens,
    idx,
    options,
    env,
    self
  ) {
    const aIndex = tokens[idx].attrIndex('target')
    if (aIndex < 0) tokens[idx].attrPush(['target', target])
    else tokens[idx].attrs[aIndex][1] = target
    return newRenderer(tokens, idx, options, env, self)
  }
}
```